scripts/common.py

```diff
@@ -57,6 +57,13 @@
 # there is a 'make test' bug in redis 6.0.6 where the solution has not been
 # released. This is explained in this issue:
 # https://github.com/redis/redis/issues/7540.
+# When upgrading the redis cli version, make sure to run 'make test' on the
+# new version to make sure that there are no issues with the program. To run
+# 'make test' download the newest version of the cli and extract it. Change
+# directories into the new folder you created titled redis-<new version>/
+# and from the top level directory of this folder, type 'make test'.
+# Make sure that ALL tests pass before making the upgrade official. If any tests
```

Comment:
... before committing the upgrade to develop.

Also shouldn't you ask them to give some indication in their PR message that they confirm these tests pass?